taxi/models.py

```diff
@@ -1,21 +1,34 @@
-from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
 
 
 class Manufacturer(models.Model):
-    name = models.CharField(max_length=255, unique=True)
-    country = models.CharField(max_length=255)
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
```

Comment:
Lots of unnecessary changes, Why did you change model file and the maxlength of fields?